.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,7 +8,7 @@ https://github.com/blog/1506-closing-issues-via-pull-requests for more
 information.
 
 ### checklist
-- [ ] Make sure you read and understood the [CONTRIBUTING.md](https://github.com/xh3b4sd/anna/.github/CONTRIBUTING.md).
+- [ ] Make sure you read and understood the [CONTRIBUTING.md](/.github/CONTRIBUTING.md).
```

Comment:
Forget what I said. I am still sleeping. You are right. I tested it by simply updating the description of this PR. There you can easily modify the link and test it in the preview mode. So what works is this link, what I do not like that much, but it works. So I will go with it. 

```
https://github.com/xh3b4sd/anna/blob/master/.github/CONTRIBUTING.md
```